showcase/opentelemetry-helper/configs/testGrpc_OperationCancelled.yaml

```diff
@@ -0,0 +1,17 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: "localhost:4320"
+
+exporters:
+  file:
+    path: "testGrpc_OperationCancelled_metrics.txt"
+
+service:
+  extensions: []
+  pipelines:
+    metrics:
+      receivers: [otlp]
+      processors: []
+      exporters: [file]
```

Comment:
These yaml files look to be all the same (barring the endpoint which looks to be incremented from ~43xx) and the file path. 

Can we consolidate this somehow? Perhaps we have one template file and the required params are spliced into the file for each exporter?

I'm don't think that having one file for each metric (i.e. attempt, operation, retry) and duplicating it for each transport is maintainable/ scalable.

Reply:
I tried, but even if we have a template file, we need some sort of scripting to create a new file using it, supplying the endpoint and file path and use it as the config for the collector. That would require invoking another script for every test. Still thinking of a better solution..

Reply:
We have one file for every test, not every metric. And for each transport, we have a similar test.

Reply:
This is fixed. We do not have these files now. We're generating a new config file for each test, and destroying it during cleanup.

Reply:
IMO, I think having multiple scripts, if needed, to create the startup logic for otel is fine (as long as we just need to call one entry point `./startup_otel.sh`). I just don't think we should have to copy and paste a new YAML file every time we want to add a new test case for otel showcase.

If we do need a new otel collector for every test, I think it would be better to have like a mapping somewhere and have the startup script read through the mapping and create the otel collectors from there. Not ideal given that we have to update both the Java unit tests and create a new entry in the script. That's why I have a bit of preference for a single otel collector if possible. If there are limitations with the single otel collector, then my preference would be not have a ton of YAML files.